CodeChat_Server/install.rst

```diff
@@ -13,48 +13,57 @@ To install the CodeChat System, install the CodeChat Server and a CodeChat exten
 
 CodeChat Server Installation
 ============================
+ .. note:
```

Comment:
This should be merged with `easyinstall.rst` so there's just one combined file named `install.rst`; delete `easyinstall.rst.`.